shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/segment/impl/comments/ExpectedComments.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.segment.impl.comments;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.segment.AbstractExpectedSQLSegment;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+/**
+ * Expected comments.
+ */
+@Getter
+@Setter
+public class ExpectedComments extends AbstractExpectedSQLSegment {
+    @XmlAttribute
```

Comment:
leave an empty line between the class clarification and XmlAttribute annotation, like :

```java
public class ExpectedComments extends AbstractExpectedSQLSegment {
    
    @XmlAttribute
```

Reply:
ok, thanks.

> leave an empty line between the class clarification and XmlAttribute annotation, like :
> 
> ```java
> public class ExpectedComments extends AbstractExpectedSQLSegment {
>     
>     @XmlAttribute
> ```

ok, thanks for reminding.